collect_app/src/main/AndroidManifest.xml

```diff
@@ -149,6 +149,9 @@ the specific language governing permissions and limitations under the License.
         <activity
             android:name=".activities.GeoPolyActivity"
             android:configChanges="orientation" />
+        <activity
+            android:name=".activities.FormMapActivity"
+            android:configChanges="orientation" />
```

Comment:
Good catch, thank you. I have removed it. It does indeed block restart on orientation change which we shouldn't need to do, especially since map center and zoom are explicitly saved in `onSaveInstanceState`.

I'm sorry this whole thing is a bit of a mess lifecycle-wise. I evidently didn't carefully go back and review everything Ping had originally done and got myself in a pretty confused state, apparently. I didn't feel confused at the time. "Unknown confusion" is the worst kind of confusion.